app/upgrades/v25/upgrades.go

```diff
@@ -63,3 +83,48 @@ func AddCelestiaValidators(ctx sdk.Context, k stakeibckeeper.Keeper) error {
 	}
 	return nil
 }
+
+// Updates the outer redemption rate bounds
+func UpdateRedemptionRateBounds(ctx sdk.Context, k stakeibckeeper.Keeper) {
+	ctx.Logger().Info("Updating redemption rate outer bounds...")
+
+	for _, hostZone := range k.GetAllHostZone(ctx) {
+		// Give osmosis a bit more slack since OSMO stakers collect real yield
+		outerAdjustment := RedemptionRateOuterMaxAdjustment
+		if hostZone.ChainId == OsmosisChainId {
+			outerAdjustment = outerAdjustment.Add(OsmosisRedemptionRateBuffer)
+		}
+
+		outerMinDelta := hostZone.RedemptionRate.Mul(RedemptionRateOuterMinAdjustment)
+		outerMaxDelta := hostZone.RedemptionRate.Mul(outerAdjustment)
+
+		outerMin := hostZone.RedemptionRate.Sub(outerMinDelta)
+		outerMax := hostZone.RedemptionRate.Add(outerMaxDelta)
+
+		hostZone.MinRedemptionRate = outerMin
+		hostZone.MaxRedemptionRate = outerMax
+
+		k.SetHostZone(ctx, hostZone)
+	}
+}
+
+// Tighten Celestia's inner bounds as a safety measure
+func UpdateCelestiaInnerBounds(ctx sdk.Context, k stakeibckeeper.Keeper) {
+	ctx.Logger().Info("Tightening Celestia inner bounds...")
+
+	hostZone, found := k.GetHostZone(ctx, staketiatypes.CelestiaChainId)
+	if !found {
+		ctx.Logger().Error("Celestia host zone not found, could not update inner bounds")
+		return
+	}
+
+	innerRedemptionRateDelta := hostZone.RedemptionRate.Mul(RedemptionRateInnerAdjustment)
+
+	minInnerRedemptionRate := hostZone.RedemptionRate.Sub(innerRedemptionRateDelta)
+	maxInnerRedemptionRate := hostZone.RedemptionRate.Add(innerRedemptionRateDelta)
```

Comment:
we meant this just as a direct adjustment, rather than a multiplier

```suggestion
	minInnerRedemptionRate := hostZone.RedemptionRate.Sub(RedemptionRateInnerAdjustment)
	maxInnerRedemptionRate := hostZone.RedemptionRate.Add(RedemptionRateInnerAdjustment)
```